index.html

```diff
@@ -1,289 +1,647 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio Mobile Version Skeleton</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200&family=Poppins:ital,wght@0,700;1,200&family=Roboto+Slab&display=swap" rel="stylesheet">
-
-</head>
-<body>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200&family=Poppins:ital,wght@0,700;1,200&family=Roboto+Slab&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
     <header class="header-style">
-        <div class="logo">
-            
-           
-           <a href="#" class="logo">MY LOGO</a>
-            
-        </div>
-
-        <nav class="navbar">
-            <img src="Union.svg" alt="Navbar-Image" class="nav-image">
-        </nav>
-
-        
-            
-        
+      <div class="logo">
+        <a href="#" class="logo">MY LOGO</a>
+      </div>
+
+      <nav class="navbar">
+        <img src="Union.svg" alt="Navbar-Image" class="nav-image" />
+      </nav>
+
+      <nav class="navbarTwo">
+        <ul class="navbarTwo-details">
+          <li class="navbarTwo-list">
+            <a class="navbarTwo-link" href="#portfolio">Portfolio</a>
+          </li>
+          <li class="navbarTwo-list">
+            <a class="navbarTwo-link" href="#about">About</a>
+          </li>
+          <li class="navbarTwo-list">
+            <a class="navbarTwo-link" href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Good job implementing the desktop design, but it seems some of the mobile designs were affected in the process. Please ensure that both mobile and desktop designs align with the original design. In this case, your navbar and headline sections does not match the design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165739104-58aef7c3-e1b0-436d-aabd-5226048d510c.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165739282-2072c4f8-7612-434d-a617-cabe4f11cbdc.png)